`usePanEvent` test sometimes fails

### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.12

### Description

```
 FAIL  packages/hooks/use-pan-event/tests/index.test.tsx > usePanEvent > should call onEnd when pan ends
AssertionError: expected "spy" to be called with arguments: [ Any<Object>, Any<Object> ]

Received: 



Number of calls: 0


Ignored nodes: comments, script, style
<html
  data-mode="light"
  data-theme="base"
  style="color-scheme: light;"
>
  <head />
  <body
    class="ui-light"
  >
    <div>
      <div
        data-testid="el"
      />
      <span
        hidden=""
        id="__ui_dev"
      />
    </div>
    <div
      class="ui-portal"
    >
      <ul
        class="ui-notice__list ui-notice__list--top css-1nx8pei"
      />
      <ul
        class="ui-notice__list ui-notice__list--top-left css-630uh8"
      />
      <ul
        class="ui-notice__list ui-notice__list--top-right css-8fufzu"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom css-oote3h"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom-left css-y29g6d"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom-right css-1qrpoux"
      />
    </div>
  </body>
</html>...
 ❯ packages/hooks/use-pan-event/tests/index.test.tsx:140:21
    138| 
    139|     await waitFor(() =>
    140|       expect(onEnd).toHaveBeenCalledWith(
       |                     ^
    141|         expect.any(Object),
    142|         expect.any(Object),
 ❯ runWithExpensiveErrorDiagnosticsDisabled node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/config.js:47:12
 ❯ checkCallback node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/wait-for.js:124:77
 ❯ Timeout.checkRealTimersCallback node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/wait-for.js:118:16

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


Error: AssertionError: expected "spy" to be called with arguments: [ Any<Object>, Any<Object> ]

Received: 



Number of calls: 0


Ignored nodes: comments, script, style
<html
  data-mode="light"
  data-theme="base"
  style="color-scheme: light;"
>
  <head />
  <body
    class="ui-light"
  >
    <div>
      <div
        data-testid="el"
      />
      <span
        hidden=""
        id="__ui_dev"
      />
    </div>
    <div
      class="ui-portal"
    >
      <ul
        class="ui-notice__list ui-notice__list--top css-1nx8pei"
      />
      <ul
        class="ui-notice__list ui-notice__list--top-left css-630uh8"
      />
      <ul
        class="ui-notice__list ui-notice__list--top-right css-8fufzu"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom css-oote3h"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom-left css-y29g6d"
      />
      <ul
        class="ui-notice__list ui-notice__list--bottom-right css-1qrpoux"
      />
    </div>
  </body>
</html>...
 ❯ packages/hooks/use-pan-event/tests/index.test.tsx:140:21
 ❯ runWithExpensiveErrorDiagnosticsDisabled node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/config.js:47:12
 ❯ checkCallback node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/wait-for.js:124:77
 ❯ Timeout.checkRealTimersCallback node_modules/.pnpm/@testing-library+dom@10.1.0/node_modules/@testing-library/dom/dist/wait-for.js:118:16


 Test Files  1 failed | 144 passed (145)
      Tests  1 failed | 1285 passed | 2 skipped | 2 todo (1290)
   Start at  10:06:01
   Duration  153.64s (transform 3.99s, setup 19.72s, collect 138.39s, tests 162.16s, environment 74.43s, prepare 17.67s)

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

### Link to Reproduction

_No response_

### Steps to Reproduce

1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error


### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

--- a/.changeset/polite-pens-joke.md
+++ b/.changeset/polite-pens-joke.md
@@ -1,0 +1,5 @@
+---
+"@yamada-ui/use-pan-event": patch
+---
+
+Fixed a bug that caused event processing to not occur due to incorrect usage of `cancelSync`.

--- a/packages/hooks/use-pan-event/src/index.ts
+++ b/packages/hooks/use-pan-event/src/index.ts
@@ -5,7 +5,7 @@ import {
   getEventPoint,
   isMultiTouchEvent,
 } from "@yamada-ui/utils"
-import sync, { cancelSync, getFrameData } from "framesync"
+import sync, { getFrameData } from "framesync"
 import type { RefObject } from "react"
 import { useEffect, useRef } from "react"
 
@@ -163,7 +163,7 @@ const panEvent = (
 
     onSessionEnd?.(ev, panInfo)
 
-    end()
+    removeListeners()
 
     if (!onEnd || !startEvent) return
 
@@ -174,21 +174,15 @@ const panEvent = (
     handlers = newHandlers
   }
 
-  let removeListeners = pipe(
+  const removeListeners = pipe(
     addPointerEvent(win, "pointermove", onPointerMove),
     addPointerEvent(win, "pointerup", onPointerUp),
     addPointerEvent(win, "pointercancel", onPointerUp),
   )
 
-  const end = () => {
-    removeListeners?.()
-
-    cancelSync.update(updatePoint)
-  }
-
   return {
     updateHandlers,
-    end,
+    removeListeners,
   }
 }
 
@@ -249,7 +243,7 @@ export const usePanEvent = (
 
   useEffect(() => {
     return () => {
-      panSession.current?.end()
+      panSession.current?.removeListeners()
       panSession.current = null
     }
   }, [])

